kubernetes/captain-hook-master-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+    name: captain-hook-master-deployment
+    labels:
+        app: captain-hook
+        tier: backend
+        maintainer: python
+spec:
+    replicas: 2
+    maxUnavailable: 1
+    revisionHistoryLimit: 3
+    selector:
+        matchLabels:
+            app: captain-hook
+    template:
+        metadata:
+            labels:
+                app: captain-hook
+        spec:
+            containers:
+                - name: captain-hook-master
+                  image: roihunter.azurecr.io/captain-hook/master:$BUILD_NUMBER
+                  imagePullPolicy: Always
+                  readinessProbe:
+                    httpGet:
+                      path: /health
+                      port: 8005
+                    initialDelaySeconds: 5
+                    periodSeconds: 10
+                  env:
+                      - name: GUNICORN_BIND
+                        value: '0.0.0.0:8005'
+                      - name: GUNICORN_WORKERS
+                        value: '1'
```

Comment:
@daniilsvirin So there will be 2 replicas each with 1 worker? There used to be 4 workers, but I don't know if it was enough or too much @martinvy

Reply:
I guess it should be enough, we can try it out with just 2 replicas, I don't see big traffic there.